docs/IRISMapCubeSequence.rst

```diff
@@ -0,0 +1,212 @@
+
+    >>> my_sequence
+    IRISMapCubeSequence
+    -------------------
+    Observatory:	 IRIS
+    Instrument:		 SJI
+
+    OBS ID:		 3690015104
+    OBS Description:	 Very large sit-and-stare 0.3x175 1s  C II   Si IV   Mg II h/k Deep x
+    OBS period:		 2018-04-26T23:07:22.780000 -- 2018-04-27T01:39:47.122000
+
+    Sequence period:	 2018-04-26T23:07:22.880000 -- 2018-04-27T01:36:40.490000
+    Sequence Shape:	 [  98. 1095. 1018.] pix
+    Axis Types:		 ('time', 'custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon')
+
+Here we can find some information about our object such as when the observation was made,
+the OBS ID, its dimensions, etc.
+
+Class Structure
+---------------
+
+Sequence Attributes
+^^^^^^^^^^^^^^^^^^^
+
+Our IRISMapCubeSequence object, here called ``my_sequence`` is now created and inspected, we can look into its structure.
+
+- ``my_sequence.data`` : In this attribute, we can find the data array of our sequence.
+- ``my_sequence.meta`` : The meta data contains a lot of information which are not inside
+  the data array.
+- ``my_sequence.common_axis``: We can found some explanation in NDCubeSequence.Common_Axis_
+  paragraph.
+
+Data
+""""
+
+The ``data`` attribute of a IRISMapCubeSequence object contains the list of all cubes
+inside the object. We can see this list by doing: ::
+
+    >>> my_sequence.data
+    [
+         IRISMapCube
+         ---------
+         Observatory:		 IRIS
+         Instrument:			 SJI
+         Bandpass:			 1330.0
+         Obs. Start:			 2018-04-26T23:07:22.780000
+         Obs. End:			 2018-04-27T01:39:47.122000
+         Instance Start:		 2018-04-26T23:07:22.880000
+         Instance End:		 2018-04-27T01:36:40.490000
+         Total Frames in Obs.:	 49
+         IRIS Obs. id:		 3690015104
+         IRIS Obs. Description:	 Very large sit-and-stare 0.3x175 1s  C II   Si IV   Mg II h/k Deep x
+         Cube dimensions:		 [  49. 1095. 1018.] pix
+         Axis Types:			 ('time', 'custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon')
+         ,
+         IRISMapCube
+         ---------
+         Observatory:		 IRIS
+         Instrument:			 SJI
+         Bandpass:			 1400.0
+         Obs. Start:			 2018-04-26T23:07:22.780000
+         Obs. End:			 2018-04-27T01:39:47.122000
+         Instance Start:		 2018-04-26T23:08:25.360000
+         Instance End:		 2018-04-27T01:37:42.990000
+         Total Frames in Obs.:	 49
+         IRIS Obs. id:		 3690015104
+         IRIS Obs. Description:	 Very large sit-and-stare 0.3x175 1s  C II   Si IV   Mg II h/k Deep x
+         Cube dimensions:		 [  49. 1095. 1018.] pix
+         Axis Types:			 ('time', 'custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon')
+         ]
+
+
+The return value is a list with the representation of each cube inside. For an upcoming work, we
+will need to select only on cube. We can do that by indexing our sequence from 0 to N-1,
+where N is the number of cubes. To select the first cube do: ::
```

Comment:
This sounds a bit confusing since although the data is split into two cubes, we've designed the slicing to allow users to treat it as one contiguous cube since the sequence axis has no physical meaning (unlike for spectrograph).  I'll need to think a little more about how best to word this.

Reply:
Yes, I've ask you before but maybe the message went to an outdated review. I was asking you if we should mix the cubes and if yes in which order ? If we use time order, the wavelength will be mixed, opposite if we use wavelength (by wavelength I mean 2 different cube like cubes I used).

Reply:
Yes this is what I said, but of course, I think that keeping the passband cubes is better. Do you want to regroup in only one cube the data ?

Reply:
No the data must physical reside in separate cubes because they have different WCS transformations.  But the user should interact with the sequence like it's a cube.

Yes, we definitely want to avoid that.

Reply:
If I summarize, you prefer creating a sequence with `explode_along_axis` than what I did, and then, keep only the same passband. So maybe we must add so code in `IRISMapCubeSequence` to use only one passband ?

Reply:
Not quite.  You can use different FITS files, so long as they are from the same passband and have the same OBS ID.  This can happen in a couple of ways, like if a single observing campaign is interrupted by an eclipse (i.e. the Earth get's between the Sun and IRIS), or if the same observing mode (OBD ID) is used for two observing campaigns, e.g. if a long term study is carried out.  But by far the more common is due to eclipses breaking up a single observing campaign into two FITS files.

```explode_along_axis``` is valid way of making a sequence out of one FITS file.

I'm not sure we need to raise an error in ```IRISMapCubeSequence``` at this stage.  Maybe as the object gets used we will find users who like to use it that way.  But I think there should be an error raised by ```read_iris_sji_level2_fits``` if there isn't already.

Reply:
Let's have the errors raised in the FITS reading function for now.